packages/core/src/modules/vc/__tests__/W3cCredentialService.test.ts

```diff
@@ -349,6 +349,11 @@ describe('W3cCredentialService', () => {
           },
         }
 
+        // FIXME: fails with:
```

Comment:
can you keep this as is? It's not related to this PR and is good to keep failing

Reply:
yes for sure. was just curious if this runs through without this test.